packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts

```diff
@@ -66,7 +66,7 @@ export class DeviceService {
 
         const irecDevice = await this.irecService.createDevice(user, {
             ...deviceData,
-            capacity: deviceData.capacity / 1e6,
+            capacity: String(Number(deviceData.capacity) / 1e6),
```

Comment:
`Number(deviceData.capacity)` will overflow if the number is too big.

We should use BigNumber here.
```js
BigNumber.from(deviceData.capacity).div(1e6)
```

Reply:
Number.MAX_SAFE_INTEGER is 9007199254740991, I think it is enough

Reply:
fixed